kubernetes/deployment-production-azure-canary.tmpl

```diff
@@ -0,0 +1,154 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: panoptes-production-azure-canary-app
```

Comment:
Not blocking - I don't think we need the azure in the name here tbh
```suggestion
  name: panoptes-production-canary-app
```

Reply:
Valid - the reasoning behind putting azure in the name was so that if we set up another canary for a different purpose in the future, there wouldn't be confusion. But most likely, at that point we will no longer be needing the azure canary for testing.